core/src/main/java/org/testcontainers/containers/wait/strategy/ShellStrategy.java

```diff
@@ -0,0 +1,29 @@
+package org.testcontainers.containers.wait.strategy;
+
+import org.rnorth.ducttape.TimeoutException;
+import org.rnorth.ducttape.unreliables.Unreliables;
```

Comment:
Can you use Awaitility instead, please?

Reply:
There is a different PR #6677 moving to Awaitility. I would take care of it once is done.